docs/Documentation/Applications/ipopt.md

```diff
@@ -40,8 +42,8 @@ We will use COIN-OR's [coinbrew](https://github.com/coin-or/coinbrew) repo to bu
     ./coinbrew build Ipopt --disable-java --prefix="${MYAPPS}" --with-metis-cflags="-I${MYINC}" --with-metis-lflags="-L${MYLIB} -lmetis" --with-lapack-lflags="-L${MKLROOT}/lib/intel64 -Wl,--no-as-needed -lmkl_rt -lpthread -lm -ldl" --with-lapack-cflags="-m64 -I${MKLROOT}/include" ADD_CFLAGS="-march=skylake-avx512" ADD_FCFLAGS="-march=skylake-avx512" ADD_FFLAGS="-march=skylake-avx512"
     ```
 
-    * `build Ipopt` tells `coinbrew` to configure and build Ipopt and its dependencies
-    * `--disable-java` says to build Ipopt without the java interface
+    * `build Ipopt` tells `coinbrew` to configure and build IPOPT and its dependencies
+    * `--disable-java` says to build IPOPT without the java interface
     * `--prefix` says to install the library in "${MYAPPS}"
     * `--with-metis-cflags` gives the compiler the location of the metis header "metis.h"
```

Comment:
MKL link in step 9 doesn't work (in "Eagle which is [recommended for perfomance reasons](https://www.nrel.gov/hpc/eagle-software-libraries-mkl.html)."

Reply:
I think it would be better to update the installation instructions for IPOPT in a different PR since it will make this  PR very large and delay publishing of information relevant to running IPOPT on Kestrel. For now, I can add a warning that points user to the IPOPT modules on Kestrel and let them know that the following instructions are for Eagle and will be updated shortly. 

Updating the installation instructions for a performant built from source IPOPT on Kestrel will require some testing that will take time.

Reply:
That works, I made issue #692 for updating the IPOPT page